### ISSUE TITLE
Boot sequence for example in 'greetings' 

### ISSUE THREAD

**Issue by davidem** [coolapso/hugo-theme-terminalcv#12]
Is it possible to create a boot sequence? So it looks like it is booting up when the page loads? 

The way I'm trying now is to tweak 'greetings', but I can't seem to add a 'clear' command between the sequence and the normal greeting. 

This is what I have now: 

`greetings: "{{ .Site.Params.bootsequence }}[[b;{{ .Site.Params.greetingBannerColor | default "white" }};]{{ .Site.Params.greetingBanner }}][[b;{{ .Site.Params.greetingColor | default "white" }};]{{ .Site.Params.greeting }}]",`

Is it possible to add a 'clear' command after the bootsequence? My jQuery/javascript skills are limited ;) 

Or is there another way?

**Comment by coolapso** [comment]
A boot sequence would be cool :D 

Would need to test that out. There is a clear command that comes by default with the jcubic/jquery-terminal, maybe you can use that one?

**Comment by davidem** [comment]
I know, it works when I type it. But if I try to put it in 'greetings' I get blank pages or errors. As said: my jquery skills are barely existent. 😉

**Comment by coolapso** [comment]
@davidem  to clear the terminal you can call the clear() function: `term.clear();`

**Comment by davidem** [comment]
Yes! I finally have what I want 😁

In layouts/index.html I added the following below 'prompt': 

```
        greetings: false,
        onInit: function(term) {
          term.pause();
          setTimeout(function() {
            term.typing('echo', 1, '{{ .Site.Params.bootsequence1 }}', function() {  });
          }, 1);
          setTimeout(function() {
            term.echo('{{ .Site.Params.bootsequence2 }}');
            term.typing('echo', 1, '{{ .Site.Params.bootsequence3 }}', function() {  });
          }, 600);
          setTimeout(function() {
            term.echo('{{ .Site.Params.bootsequence4 }}');
            term.typing('echo', 1, '{{ .Site.Params.bootsequence5 }}', function() {  });
          }, 1200);
          setTimeout(function() {
            term.echo('{{ .Site.Params.bootsequence6 }}');
            term.typing('echo', 1, '{{ .Site.Params.bootsequence7 }}', function() {  });
          }, 1600);
          setTimeout(function() {
            term.echo('{{ .Site.Params.bootsequence8 }}');
            term.typing('echo', 1, '{{ .Site.Params.bootsequence9 }}', function() {  });
          }, 2000);
          setTimeout(function() {
            term.clear();
          }, 3000);
          setTimeout(function() {
            term.echo("[[b;{{ .Site.Params.greetingBannerColor | default "red" }};]{{ .Site.Params.greetingBanner }}]");
            term.typing('echo', 1, "[[b;{{ .Site.Params.greetingColor | default "green" }};]{{ .Site.Params.greeting }}]", function() {  });
            term.resume();
          }, 4000);
        },
```

The bootsequence (I currently use [this](https://gist.github.com/MattDahEpic/bc26e996247242173242351e7d8e5477) as an example) I placed in hugo.yaml and split it up to mimic loading time difference. I noticed the timings need to be precise, otherwise the echo's and typings get mixed up. Hence the multiple bootsequence functions. 

After a brief moment the screen gets cleared with term.clear() (thanks!), a banner is show and then the normal greeting. 

It doesn't look very pretty, but it works. 

As it is very specific to my needs, I doubt you want me to create a pull request. But if you do, let me know 😁


Thanks!

**Comment by coolapso** [comment]
I just made a huge refactor on the whole template and released v2.0.0 ... 
if you want to make a PR to add the boot sequence I am fine with it ... just make sure it has a feature gate to be enabled /disabled
